benchmarks/time_forest_partition.cxx

```diff
@@ -339,6 +346,8 @@ t8_time_forest_create_cmesh (const char *msh_file, int mesh_dim,
                                           T8_LOAD_STRIDE, stride);
     /* Partition only if more than 1 input file */
     partition = num_files > 1;
+    /* Abort if use_occ option selected but no msh file provided */
```

Comment:
This check should be placed [here](https://github.com/DLR-AMR/t8code/blob/3f73e1715f9a3c76ed7a8c03f0e00a14bdcfb846/benchmarks/time_forest_partition.cxx#L469-L477). The checks for wron user input are always handled directly after the parsing of the CLI options.